frontend/packages/data-portal/app/components/Filters/BooleanFilter.tsx

```diff
@@ -16,7 +16,7 @@ export function BooleanFilter({
   value: boolean
 }) {
   return (
-    <div className={cns('pl-sds-m whitespace-nowrap', styles.boolean)}>
+    <div className={cns('pt-sds-m pl-sds-m whitespace-nowrap', styles.boolean)}>
```

Comment:
was the switch from left to top intentional?

Reply:
@codemonkey800 For the type size, this one is a customization from the checkbox component so that it is the same as the filter dropdowns (hierarchy thing). font size should be `fontBodyS`

I believe it should have top padding, but we should also ensure the padding looks even with the other filters in that section (currently too much space between ground truth check box and "available files in the dev link) -- cc @kev-zunshiwang 

<img width="226" alt="Screenshot 2024-08-28 at 2 12 56 PM" src="https://github.com/user-attachments/assets/e18d2e5a-63ca-48cc-a78f-6854e599b6f5">